apify-api/openapi/paths/datasets/datasets@{datasetId}@statistics.yaml

```diff
@@ -0,0 +1,49 @@
+get:
+  tags:
+    - Datasets/Statistics
+  summary: Get dataset statistics
+  description: |
+    Returns statistics for given dataset.
+    Currently provides only [field statistics](https://docs.apify.com/platform/actors/development/actor-definition/dataset-schema/validation#dataset-field-statistics).
+
+  operationId: dataset_statistics_get
+  parameters:
+    - name: datasetId
+      in: path
+      description: Dataset ID or `username~dataset-name`.
+      required: true
+      style: simple
+      schema:
+        type: string
+        example: WkzbQMuFYuamGv3YF
+    -   name: token
+        in: query
+        description: |
+            API authentication token. It is required only when using the `username~dataset-name` format for `datasetId`.
+        style: form
+        explode: true
+        schema:
+            type: string
+            example: soSkq9ekdmfOslopH
```

Comment:
@mtrunkat @netmilk - @drobnikj suggest in [this PR](https://github.com/apify/apify-core/pull/18862#discussion_r1899409785) to require the authentication always in this endpoint. Is it right?

> This is questionable as we plan to remove anonymous access for storage. We can do it for new endpoints already.
I would make this isAuthenticationRequired: true, and restrict this endpoint.

```suggestion
```

Reply:
Good point! Let's make it authenticated = less work in the next months. As we will turn all endpoints this way.